src/_guides/language/language-tour.md

```diff
@@ -3545,6 +3540,14 @@ Keep in mind the following when you use deferred loading:
   using <code>deferred as <em>namespace</em></code>.
   The `loadLibrary()` function returns a [Future](/guides/libraries/library-tour#future).
 
+<aside class="alert alert-warning" markdown="1">
+**Dart VM difference:**
+Currently, the Dart VM allows access to members of deferred libraries
+even before the call to `loadLibrary()`.
```

Comment:
maybe say something to warn about the risk? For example: "use carefully, as this will change in the near future and checks will be enforced once the bug is fixed"?

Reply:
Yeah, I vacillated about that. The "currently" is not a strong enough hint, I guess. :)